packages/web/src/components/admin_page/feature_switch_page/render_switch.jsx

```diff
@@ -0,0 +1,158 @@
+/* eslint-disable no-unused-vars */
+import React, { useState, useEffect } from 'react';
+import { Button } from 'react-bootstrap';
+import Validate from '../../../Global';
+import InvalidClient from '../invalid_client';
+import { useNavigate } from 'react-router-dom';
+import axios from 'axios';
+import { parseToNewAPI, featureSwitchesURL } from '../../../Global';
+import FeatureRow from './row_switch';
+import { OverlayTrigger, Popover, Form, Alert } from 'react-bootstrap';
+import { FsContext } from '../../../feature_switches';
+import { useContext } from 'react';
+
+const popover = (onUpdate, errorMessage) => {
+    return (
+        <Popover id="popover-basic">
+            <Popover.Header as="h3">Want to update or remove?</Popover.Header>
+            <Popover.Body>
+                {errorMessage !== undefined ? <Alert variant="warning">{errorMessage}</Alert> : null}
+                <UpdateSwitch onUpdate={onUpdate} />
+            </Popover.Body>
+        </Popover>
+    );
+};
+const UpdateSwitch = ({ onUpdate }) => {
+    const [newSwitch, setNewSwitch] = useState({
+        is_enabled: 'true',
```

Comment:
Why is the state hardcoded? Aren't you supposed to load it from the switch you're rendering? Do you even need to create a new state in children components if you have the context loaded in the parent component? Why don't you drill down the function for setting the state of switches and just use to update the appropriate switches.